opentreemap/treemap/templatetags/form_extras.py

```diff
@@ -386,7 +386,7 @@ def _field_value(model, field_name, data_type):
             units = get_units_if_convertible(
                 model.instance, object_name, field_name)
             if units != '':
-                units = get_unit_name(units)
+                units = get_unit_abbreviation(units)
```

Comment:
The width only affects layout of the diameter/circumference, which is the only place where we have side-by-side form layout. Can we add an option for abbreviation or full, and default to full?